sql/src/main/java/io/crate/metadata/cluster/DDLClusterStateModifier.java

```diff
@@ -80,4 +82,14 @@ default ClusterState onRenameTable(ClusterState currentState,
                                        boolean isPartitionedTable) {
         return currentState;
     }
+
+    /**
+     * Called while a view is dropped.
+     */
+    default ClusterState onDropView(ClusterState currentState, List<RelationName> relationNames) {
+        for (RelationName relationName : relationNames) {
+            currentState = onDropTable(currentState, relationName);
```

Comment:
I'm not so happy with defaulting this as only the implementation knows if it can handle `onDropView` the same like `onDropTable`. I'd move this to the implementing class `UserManagerDDLModifier`.